### ISSUE TITLE
<autogenerated-issue>

### ISSUE THREAD

**Issue by kpy3** [DeterminateSystems/nix-installer#1170]
## Error
```
Error: 
   0: Planner error
   1: Error executing action
   2: Action `create_group` errored
   3: Group `nixbld` existed but had a different gid (30000) than planned (350)
```

## Metadata
|key|value|
|--|--|
|**prev. version**|0.17.0|
|**version**|0.26.0|
|**os**|macos|
|**arch**|aarch64|

**Comment by kpy3** [comment]
My journey start from https://determinate.systems/posts/nix-support-for-macos-sequoia/ as preparation for MacOS upgrade. But something goes wrong, although repairer finished successfully, user ids doesn't change, and I got message about to uninstall and install nix again. So when I uninstalled 0.17.0, and try to install 0.26.0, I got error messages about existed group and number of existed users with wrong ids. 
I removed them with 
```
sudo dscl . -delete /Groups/nixbld
sudo dscl . -delete /Users/_nixbld1
...
sudo dscl . -delete /Users/_nixbld32
```
and finally able to install 0.26.0 to the my system.
